Sources/CacheAdvance/ObjectiveC.swift

```diff
@@ -51,5 +53,6 @@ struct ObjectiveCTryFailure: Error {}
 
 /// A `throw`able NSException.
 struct ObjectiveCError: Error {
-    let exception: NSException
+    let exceptionName: NSExceptionName
+    let reason: String?
```

Comment:
`ObjectiveCError` is considered `Sendable` because `Error` inherits from `Sendable`, and `ObjectiveCError` inherits from `Error`.

`Sendable` requires every member of the conforming type to also be `Sendable` or we get a warning. This change eliminates the warning by ensuring all our members are `Sendable`-conforming types.

We could have alternatively marked this type as `@unchecked Sendable`, but it seemed simpler & better to just make this type truly conform to `Sendable`

Reply:
Thanks for explaining!